setup.py

```diff
@@ -29,7 +29,6 @@ def get_version():
 
 setup(
     name='jwplatform',
-    version=get_version(),
```

Comment:
`get_version()` should be updated to bring the version from the `version.py`

Reply:
`version_file = read_file('jwplatform', '__init__.py')` should be updated to `version_file = read_file('jwplatform', 'version.py')` at 
https://github.com/jwplayer/jwplatform-py/blob/58a225892ae19edf881417cab283626be362da63/setup.py#L22